### DISCUSSION TITLE
[Bug]: How to Solve [RuntimeError: Trying to resize storage that is not resizable]

### DISCUSSION THREAD

**Discussion by Osasbro** [open-edge-platform/anomalib#1904] [Q&A]
### Describe the bug

![Screenshot 2024-03-14 160400](https://github.com/openvinotoolkit/anomalib/assets/87775707/a65e116a-386d-482e-9472-d1e8b382e105)
![Screenshot 2024-03-14 160530](https://github.com/openvinotoolkit/anomalib/assets/87775707/d3393625-8211-4e13-a7a9-56d93d3aabad)
![Screenshot 2024-03-14 160623](https://github.com/openvinotoolkit/anomalib/assets/87775707/f77d408e-a5d5-4b17-ae51-395d115c593e)
RuntimeError: Caught RuntimeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\torch\utils\data\_utils\worker.py", line 308, in _worker_loop
    data = fetcher.fetch(index)
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\torch\utils\data\_utils\fetch.py", line 54, in fetch
    return self.collate_fn(data)
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\anomalib\data\base\datamodule.py", line 46, in collate_fn
    out_dict.update({key: default_collate([item[key] for item in batch]) for key in elem})
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\anomalib\data\base\datamodule.py", line 46, in <dictcomp>
    out_dict.update({key: default_collate([item[key] for item in batch]) for key in elem})
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\torch\utils\data\_utils\collate.py", line 265, in default_collate
    return collate(batch, collate_fn_map=default_collate_fn_map)
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\torch\utils\data\_utils\collate.py", line 123, in collate
    return collate_fn_map[collate_type](batch, collate_fn_map=collate_fn_map)
  File "C:\Users\bingq\anaconda3\envs\anomalib_env\lib\site-packages\torch\utils\data\_utils\collate.py", line 161, in collate_tensor_fn
    out = elem.new(storage).resize_(len(batch), *list(elem.size()))
RuntimeError: Trying to resize storage that is not resizable

### Dataset

Folder

### Model

PADiM

### Steps to reproduce the behavior

-

### OS information

OS information:
- OS: Win11
- Python version: 3.10.0
- Anomalib version: 1.0.0
- CPU
- I'm using a custom dataset

### Expected behavior

-

### Pip/GitHub

pip

### What version/branch did you use?

-

### Configuration YAML

```yaml
-
```

### Logs

```shell
-
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by snoopybingo** [comment]
the same question too!

**Comment by samet-akcay** [comment]
I cannot reproduce this unfortunately. When I try the same configuration as yours, mine works
<img width="1169" alt="image" src="https://github.com/openvinotoolkit/anomalib/assets/17574420/31c64b3c-dd11-42d2-bdb6-956db13bcf8f">

**Comment by Osasbro** [comment]
I understand that the 'hazelnut' dataset works fine, but when I try to use my own dataset, it doesn't work. I just want to know what is different about my data that is causing this issue.

**Comment by samet-akcay** [comment]
I'll try to use MVTec images with `Folder` dataset to see if it works.

**Comment by samet-akcay** [comment]
I also tried MVTec bottle category using the `Folder` datamodule and managed to train a model. 
<img width="1224" alt="image" src="https://github.com/openvinotoolkit/anomalib/assets/17574420/3e34d8cd-64ec-4596-a339-93dace53b49f">

Not sure about what sort of images you have in your dataset

**Comment by samet-akcay** [comment]
I'm converting this to a Q&A in discussion as it does not seem to be an issue. Feel free to convert it back to an issue if there is more evidence showing that this is a bug
